pkg/pilot/cassandra/v3/pilot.go

```diff
@@ -95,6 +110,30 @@ func (p *Pilot) syncFunc(pilot *v1alpha1.Pilot) error {
 	return nil
 }
 
+func run(args ...string) error {
+	cmd := exec.Command(args[0], args[1:]...)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
```

Comment:
I think we should capture the stdout / stderr here and also add it to the error that we return.
Otherwise I think it'll be difficult to diagnose decommission failures.